docs/content/en/blog/news/may-2022-update.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2022-06-01
+title: "May 2022 update"
+linkTitle: "May 2022 update"
+weight: 993
+description: "Development status update to recap what happened last few months"
+author: Khanh Tran ([@khanhtc1202](https://twitter.com/khanhtc1202))
+---
+
+_Published by the PipeCD dev team every few months, this update will provide you with news and updates about the project! Please click [here](/blog/2022/02/10/february-2022-update/) if you want to see the last status update._
+
+### What's changed
+---
+
+It's been a while from last update about the development of our PipeCD project, but you can be assured, our PipeCD development is going well. Since the last report, PipeCD team has introduced 5 major release (from [v0.27.0](#) to the latest [v0.32.3](#)) with serveral UI improvements and some interesting features that I can wait to show you now. This blog post walks you through some of notable changes. For all other changes, please check out each release note.
```

Comment:
```suggestion
It's been a while from the last update about the development of our PipeCD project, but you can be assured that our PipeCD development is going well. Since the last report, PipeCD team has introduced 5 major releases (from [v0.27.0](#) to the latest [v0.32.3](#)) with several UI improvements and some interesting features that I can't wait to show you now. This blog post walks you through some notable changes. For all other changes, please check out each release note.
```